packages/radio-player/index.js

```diff
@@ -0,0 +1,16 @@
+import RadioPlayer from './lib/src/radio-player.js';
```

Comment:
question: what is the difference between this `index.js` and `index.d.ts`?

Reply:
The `index.js` file is the entry point for the main `radio-player` module. `index.d.ts` is the entry point for the [typescript declarations](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html), which provide the typescript types for any typescript consumers. 

When you distribute a typescript project, it first gets compiled to javascript so you lose all of the typing. If you're using javascript to consume the module, it doesn't matter since you don't have types, but if you're using typescript to consume the module, it's nice to have the type info and that's what the `.d.ts` provides. It's basically the "header file" for the javascript.